### PR TITLE
fix border-radius on radio-switch

### PR DESCRIPTION
**Pull request by jonnysp**
### Description

add radio-switch sample and fix border-radius on radio-switch when roundet disabled

Screenshot 2023-07-10 083144

### Type of changes

- [x] Bug fix (non-breaking change which fixes an issue)

### Checklist

- [ ] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)
- [ ] My code follows the code style of the project _(using `npm run lint`)_
- [ ] My change introduces changes to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed

#### Live previews

- <https://deploy-preview-38884--twbs-bootstrap.netlify.app/docs/5.3/forms/checks-radios/#switches>

### CODE REVIEW

File: scss/forms/_form-check.scss [hunk 1]
```diff
@@ -130,7 +130,12 @@
     margin-left: $form-switch-padding-start * -1;
     background-image: var(--#{$prefix}form-switch-bg);
     background-position: left center;
-    @include border-radius($form-switch-border-radius);
+    @if $enable-rounded {
```
Comment (julien-deramond) [inline scss/forms/_form-check.scss:133]: I haven't tested it, I'll make a more thorough review later, but the mixin definition is the following:

```scss
@mixin border-radius($radius: $border-radius, $fallback-border-radius: false) {
  @if $enable-rounded {
    border-radius: valid-radius($radius);
  }
  @else if $fallback-border-radius != false {
    border-radius: $fallback-border-radius;
  }
}
```

Just by reading the code, a simple `@include border-radius($form-switch-border-radius, 0);` could maybe work 🤷

Comment (jonnysp) [reply scss/forms/_form-check.scss:133]: does the same, only simpler.

File: scss/forms/_form-check.scss [hunk 2]
```diff
@@ -130,7 +130,7 @@
     margin-left: $form-switch-padding-start * -1;
     background-image: var(--#{$prefix}form-switch-bg);
     background-position: left center;
-    @include border-radius($form-switch-border-radius);
+    @include border-radius($form-switch-border-radius,0);
```
Comment (julien-deramond) [inline scss/forms/_form-check.scss:133]:
```suggestion
    @include border-radius($form-switch-border-radius, 0);
```

File: site/content/docs/5.3/forms/checks-radios.md [hunk 3]
```diff
@@ -129,6 +129,22 @@ A switch has the markup of a custom checkbox but uses the `.form-switch` class t
 </div>
 {{< /example >}}
 
+{{< example >}}
```
[the remark this answers is not in the document]
Comment (patrickhlauke) [reply site/content/docs/5.3/forms/checks-radios.md:132]: I'll admit that, subjectively, as a user I'd be very confused seeing a set of seemingly independent switch controls that in fact act as a set of radio buttons. As such, I would personally not be in favour of officially including this to our core.

[the remark this answers is not in the document]
Comment (jonnysp) [reply site/content/docs/5.3/forms/checks-radios.md]: example is removed

**Review by julien-deramond — APPROVED** [review]
LGTM!

**Comment by jonnysp** [comment]
👍

**Pull request merged**